### ISSUE TITLE
Set to private in config.php (found un users folder) and its still public. 

### ISSUE THREAD

**Issue by berlinczyk** [YOURLS/YOURLS#2181]
What was not to understand in last ticket?
I set in config.php YOURLS to be private but it is still open for everyone.
Everyone can use it which I don't want. (http://erfr.eu.org) 
Set to private in config.php (found un users folder) and its still public.
Installed on subdomain but all is working properly besides this issue

**Comment by ozh** [comment]
What's not to understand in last ticket? As I suspected, you're mixing up 2 concepts, ie "private vs public" which refer to the admin area, and the fact that you created a public interface which is, as its name suggests, public. Also, just because an issue is closed doesn't mean you cannot continue commenting, please just don't open a new issue for every reply. Thanks.

Regarding your issue, nothing that a quick Google would not answer: https://github.com/YOURLS/YOURLS/wiki/Private-Or-Public is the first result for https://encrypted.google.com/search?q=yourls+private+public

Feel free to comment *here* if something isn't clear or doesn't work for you regarding this issue.

**Comment by berlinczyk** [comment]
Now I see the possibility to write, in the last closed ticket was nothing there to comment ;o)
I red all this, also from google and in the config.php is set to private:
<?php
// Private:
define( 'YOURLS_PRIVATE', true );
And still it's open for everyone (public)
Where else can I change this?

You write, I have have created is public, when? by installing? i haven't seen such a question or so. And even if there is certain a possibility to change this, or?

Thank You for helping me!

**Comment by ozh** [comment]
Let me rephrase : if you don't want your install to be public, then **don't make a public interface** ...

**Comment by berlinczyk** [comment]
Don't You recognize, with respect, that i don't know in which moment should I have dicided about this, instaed to repeet other and other the same You could give simple advise to change this even if i had to reintstall again
